data/data/powervs/variables-powervs.tf

```diff
@@ -149,3 +149,6 @@ variable "powervs_cis_crn" {
   description = "The CRN of CIS instance to use."
 }
 
+# required for `gather bootstrap` command
+variable "bootstrap_ip" { default = "" }
+variable "control_plane_ips" { default = [] }
```

Comment:
does this mean that previously the `gather bootstrap` command was broken?

Reply:
No, they are able to find them as long as we `output` them. But TF itself probably doesn't like "undeclared" variables `output`'d out of blue??

Reply:
ah ok. can you create a subsection title for those to make a clear distinction between the setup and output vars?

Reply:
like the others:
```
################################################################
# Output Variables
################################################################
```

Reply:
Addressed in the newest force-push.